api/server.py

```diff
@@ -1,12 +1,62 @@
-from flask import Flask, jsonify
+from flask import Flask, jsonify, request, abort
+from flask_cors import CORS
+from util.config import REACT_PORT, API_PORT
 
 app = Flask(__name__)
+CORS(app, resources={r"/*": {"origins": "http://localhost:{}".format(REACT_PORT)}})
 
 
 def run(port, hostname, debug_mode):
     app.run(host=hostname, port=port, debug=debug_mode)
 
 
-@app.route("/")
-def hello_world():
+# TODO: If meta-paths for A and B will be written in Java, they will need this information in Java
+@app.route("/node-sets", methods=["POST"])
+def receive_node_sets():
+    # TODO: Check if necessary information is in request object
+    if not request.json:
+        abort(400)
+
+
+@app.route("/node-sets", methods=["GET"])
+def send_node_sets():
```

Comment:
I don't like the naming yet. Send would implicate that this function sends something somewhere else async. But actually someone is pulling information, right? How about actually naming it `get_node_sets`?